pkg/image/apiserver/registry/image/strategy.go

```diff
@@ -99,7 +100,7 @@ func (s imageStrategy) PrepareForUpdate(ctx context.Context, obj, old runtime.Ob
 	if newImage.DockerImageManifest != oldImage.DockerImageManifest {
 		ok := true
 		if len(newImage.DockerImageManifest) > 0 {
-			ok, err = util.ManifestMatchesImage(oldImage, []byte(newImage.DockerImageManifest))
+			ok, err = internalimageutil.ManifestMatchesImage(oldImage, []byte(newImage.DockerImageManifest))
```

Comment:
Dang, I'm gonna needs those somehow to work with external stuff to make the describers work :disappointed: